test/views/components/PageSpec.scala

```diff
@@ -59,7 +59,7 @@ class PageSpec extends WordSpec with Matchers with GuiceOneAppPerSuite {
     val bulletPointList = BulletPointList(bulletPointLeadingText,
       Seq( bulletPointOne, bulletPointTwo, bulletPointThree ) )
 
-    val simplePage =  Page("root", Seq(para, H1(title), bulletPointList ) )
+    val simplePage =  StandardPage("root", Seq(para, H1(title), bulletPointList ) )
```

Comment:
Just a minor point, but should the H1 element be the first element in the sequence?

Reply:
This can be left as it is.